tests/test_client.py

```diff
@@ -357,67 +357,48 @@ def test_environment_caching(self):
             # is cached.
             self.assertTrue(client._in_gae_environment())
 
-    def _environment_check_gce_helper(self, status_ok=True, socket_error=False,
+    def _environment_check_gce_helper(self, status_ok=True,
                                       server_software=''):
-        response = mock.Mock()
         if status_ok:
-            response.status = http_client.OK
-            response.getheader = mock.Mock(
-                name='getheader',
-                return_value=client._DESIRED_METADATA_FLAVOR)
+            headers = {
```

Comment:
Can this be a module-scope constant? Does placing it at module-scope save having to line-wrap one of its key-value pairs?